microk8s-resources/wrappers/run-kubelite-with-args

```diff
@@ -108,18 +108,14 @@ then
 fi
 
 #UFW configuration
```

Comment:
I see with this PR we are also changing our UFW approach. We try to find UFW from the host and remove it from the snap. Could we please have this in a separate PR with justification on why this is better?

Reply:
reverted